src/components/UnivAutoCompleteInput/UnivAutoCompleteInput.css.ts

```
@@ -0,0 +1,56 @@
+import { colors } from '@/styles/colors';
+import { fonts } from '@/styles/fonts.css';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)